bitbots_vision/src/bitbots_vision/vision_modules/field_boundary.py

```diff
@@ -68,14 +68,38 @@ def set_image(self, image):
         self._convex_field_boundary_points = None
         self._mask = None
 
-    def get_mask(self):
+    def get_mask(self, offset=0):
         # type: () -> np.array
         """
+        :param offest: A vertical field boundary offset shift
         :return: np.array
         """
         # Compute mask (cached)
         self._compute_mask()
-        return self._mask
+        return self._shift_field_boundary_mask(self._mask, offset)
+
+    def _shift_field_boundary_mask(self, mask, offset):
+        shape = mask.shape
+
+        if offset == 0:
+            return mask
+
+        elif offset < 0:
+            # Shift mask downwards
+            offset = min(-offset, shape[0]-1)
+            frame = np.zeros(shape, dtype=np.uint8)
```

Comment:
Theoretically, we could shift the matrix in place and fill the missing stripe with zeros. This should increase the performance quite a lot.
(Moved from other file)

Reply:
Practically, we could not shift the matrix, since, in my testing, it is not faster.